history-network.md

```diff
@@ -179,8 +181,21 @@ HistoricalRootsBlockProof = Container[
     slot: Slot
 ]
 
-BlockHeaderProof = Union[None, AccumulatorProof, HistoricalRootsBlockProof]
+# BeaconState historical_summaries proof (for Capella onwards)
+HistoricalSummariesProof = Vector[Bytes32, 13] # Proof that BeaconBlockHeader root is part of HistoricalSummaries
```

Comment:
Yes I know, I mentioned that name clash here: https://github.com/ethereum/portal-network-specs/pull/287#discussion_r1569186217

I might reverse the naming because of that.

Reply:
I've altered the naming which should avoid the name clash and hopefully also be more clear.